spec/models/event/ical_spec.rb

```diff
@@ -0,0 +1,26 @@
+# encoding: utf-8
+
+#  Copyright (c) 2012-2013, Jungwacht Blauring Schweiz. This file is part of
+#  hitobito and licensed under the Affero General Public License version 3
+#  or later. See the COPYING file at the top-level directory or at
+#  https://github.com/hitobito/hitobito.
+
+require 'spec_helper'
+
+describe Event do
```

Comment:
I would put this in the main `event_spec.rb` file. The other files in the directory are separate classes, not aspects. Putting this spec in the main event-spec would keep the structure consistent with the convention.